### PR TITLE
Design Recent Work And About Me Section

### PR DESCRIPTION
**Pull request by waseemqureshi2020**
* Add My Recent Work Section.
* Add About Me Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,28 +12,215 @@
...
-    </section>
+    <section class="work-section">
+        <div class="work-container">
+            <div><h1 class="recent-work">My Recent Works</h1></div>
+       <div class="work-image">
+           <img src="images/My Work Images/Img Placeholder.png" alt="work">
+           </div>
+           <div class="work-content">
+            <h2 class="post-type1">Multi-Post Stories</h2>
+            <p class="para-type1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
```
Comment (ghost) [inline index.html:49]: - [ ] Kindly give your `multi-post stories` section a margin to match the Figma design `hint`.  use `margin-left`  and `margin-right`.

image
 
### Figma Design

image

- [ ] Kindly style your buttons border to match the figma design.  You can use `border: 1px solid color`

image

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @waseemqureshi2020 , 

Excellent job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)

## Highlights

## Highlights:
✅ Correct use of Github workflow
✅ No linter error
✅ Design looks good 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (waseemqureshi2020) [reply index.html:49]: I have changed CSS properties for both of these sections, Kindly have a look at the design now.

**Review by BertrandConxy — APPROVED** [review]
Hi @waseemqureshi2020, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html:49]: Good work @waseemqureshi2020 for making the changes👍 . I see it has already been approved

**Pull request merged**